_posts/function-contracts.md

```diff
@@ -0,0 +1,298 @@
+# Function Contracts for Kani
+
+I this blogpost we discuss a new feature we’re developing for Kani: Function Contracts. It’s now available as an unstable feature with the `-Zfunction-contracts` flag. If you would like to learn more about the development and implementation details of this feature and leave feedback please refer to [the RFC](https://model-checking.github.io/kani/rfc/rfcs/0009-function-contracts.html).
+
+## Introduction
+
+Verification is a costly art. If you are an active user of Kani you know that the runtime of a proof does not scale linearly with the amount of code under verification and you probably have a number of functions and ideas for harnesses you’d like Kani to verify, but it just takes prohibitively long.
```

Comment:
I think you can rewrite this sentence in a way that you are saying something good about function contracts, instead of saying something bad about the current state-of-the-art.